osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -144,7 +144,7 @@ private void load(OverlayColourProvider colourProvider, DrawableComment? parentC
                                             Size = new Vector2(avatar_size),
                                             Children = new Drawable[]
                                             {
-                                                new UpdateableAvatar(Comment.User)
```

Comment:
Value of `showUserPanelOnHover` changed, intended?

Reply:
If my answer matters, then it did change, to be the same as the web behavior.